src/sw/Core/Inc/SensorObserver.h

```diff
@@ -0,0 +1,10 @@
+#ifndef _SENSOROBSERVER_H_
+#define _SENSOROBSERVER_H_
+
+class SensorObserver
+{
+	public:
+		virtual int update() = 0;
```

Comment:
No update is needed here. The listener should only notify the method where it will be reacting for sensor change.

Reply:
fixed